src/Fixtures/index.js

```diff
@@ -843,506 +780,464 @@ export default {
-      minimalPlayersNumber: 2,
-      equipmentLabel: '',
-      equipment: 'Equipment needs for the drill',
+      author: 'Moby',
+      title: 'Dwarf duel',
+      image: 'https://i1.wp.com/www.newdelhitimes.com/wp-content/uploads/2019/07/AP19204604544330.jpg?w=1024&ssl=1',
+      description: 'Duel without jumping',
+      minimalPlayersNumber: 3,
+      equipmentLabel: EquipmentLabels.BASIC,
+      equipment: '1 Disc - 2 Cones',
       durationInMinutes: 10,
-      intensity: Intensities.LOW,
-      goals: [GoalsFrisbee.DEFENSE],
-      seasonTiming: '',
-      level: Levels.BEGINNER,
-      /* video:, */
+      intensity: Intensities.MODERATE,
+      goals: [GoalsFrisbee.CUTTING],
+      seasonTiming: SeasonTimings.ANYTIME,
+      level: Levels.INTERMEDIATE,
       steps: [
         {
           id: 1,
           title: '',
           illustrationType: IllustrationType.ANIMATION,
           illustrationSource: animationSquare,
           repetition: '1',
-          rest: '90s',
-          instruction: 'Step 1: Introduce the notion of continuity',
+          rest: '',
+          instruction: '',
         },
       ],
     },
     {
       id: 1021,
       type: DrillTypes.TECHNICAL,
-      author: 'Author',
-      title: 'Frisbee Drill Title',
-      image:
-        'https://www.plu.edu/recreations/wp-content/uploads/sites/197/2019/04/ultimate-frisbee-3-2-19-1122-scaled-1536x1163.jpg',
-      description:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec scelerisque dignissim felis, at tristique mi efficitur a. Nulla pellentesque odio nunc, ut sodales ex pulvinar at. In hac habitasse platea dictumst. Pellentesque sodales nisl lorem, ac lacinia nisl lacinia eu. Proin lacinia viverra mauris, et pharetra ipsum tempus eget. Ut maximus sapien in hendrerit ultricies. Maecenas vel diam tincidunt, consequat nunc a, mattis eros. Curabitur a eros a nulla malesuada gravida. Donec massa quam, suscipit eget efficitur et, euismod sed nulla. Morbi mattis et magna a aliquam. Fusce pellentesque vel erat eget',
-      minimalPlayersNumber: 2,
-      equipmentLabel: '',
-      equipment: 'Equipment needs for the drill',
-      durationInMinutes: 20,
-      intensity: Intensities.LOW,
-      goals: [GoalsFrisbee.DEFENSE],
-      seasonTiming: '',
-      level: Levels.BEGINNER,
-      /* video:, */
+      author: 'Mooncatcher',
+      title: 'Dishie Chain',
+      image: 'https://i1.wp.com/www.newdelhitimes.com/wp-content/uploads/2019/07/AP19204604544330.jpg?w=1024&ssl=1',
+      description: 'Basic long passes drill. You want your players to alternate between throwing, running and passing',
+      minimalPlayersNumber: 12,
+      equipmentLabel: EquipmentLabels.BASIC,
+      equipment: '1 Disc - 2 Cones',
+      durationInMinutes: 10,
+      intensity: Intensities.MODERATE,
+      goals: [GoalsFrisbee.THROWING, GoalsFrisbee.CUTTING],
+      seasonTiming: SeasonTimings.ANYTIME,
+      level: Levels.INTERMEDIATE,
```

Comment:
It looks like they are all INTERMEDIATE, maybe some of them could be moved to BEGINNER or ADVANCED ?

(Then the filters are more usable, better for a demo)